models/index.js

```diff
@@ -1,7 +1,11 @@
 import User from "./Users";
 import Products from "./Products";
+import Importer from "./importer";
+import DirWatcher from "./dirwatcher";
 
 module.exports = {
   User,
-  Products
+  Products,
+  Importer,
+  DirWatcher
 };
```

Comment:
Could you use ES6 import and export in this file and Importer.js and Dirwatcher.js instead, please.